src/Mapbender/PrintBundle/Resources/public/mapbender.element.imageExport.js

```diff
@@ -88,95 +94,50 @@
-                    if (!feature.onScreen(true)) continue
-                        var geometry = geojsonFormat.extract.geometry.apply(geojsonFormat, [feature.geometry]);
+            var data = {
+                requests: printConfigs,
+                format: format,
+                width: mapSize[0],
+                height: mapSize[1],
+                centerx: mapCenter[0],
+                centery: mapCenter[1],
+                extentwidth: this.model.getWidthOfExtent(mapExtent),
+                extentheight: this.model.getHeigthOfExtent(mapExtent),
+                vectorLayers: vectorLayers
+            };
 
-                        if(feature.style !== null){
-                            geometry.style = feature.style;
-                        }else{
-                            geometry.style = layer.styleMap.createSymbolizer(feature,feature.renderIntent);
-                        }
-                        // only visible features
-                        if(geometry.style.fillOpacity > 0 && geometry.style.strokeOpacity > 0){
-                            geometries.push(geometry);
-                        } else if (geometry.style.label !== undefined){
-                            geometries.push(geometry);
-                        }
-                }
-
-                var lyrConf = {
-                    type: 'GeoJSON+Style',
-                    opacity: 1,
-                    geometries: geometries
-                };
-
-                vectorLayers.push(JSON.stringify(lyrConf))
-            }
-
-            var mapExtent = this.map.map.olMap.getExtent();
+            var url = Mapbender.configuration.application.urls.element + '/' + this.element.attr('id') + '/export';
 
-            if(num === 0){
-                Mapbender.info(Mapbender.trans("mb.print.imageexport.info.noactivelayer"));
-            }else{
-                var url = Mapbender.configuration.application.urls.element + '/' + this.element.attr('id') + '/export';
-                var format = $("input[name='imageformat']:checked").val();
-
-                var data = {
-                    requests: layers,
-                    format: format,
-                    width: imageSize.w,
-                    height: imageSize.h,
-                    centerx: mapExtent.getCenterLonLat().lon,
-                    centery: mapExtent.getCenterLonLat().lat,
-                    extentwidth: mapExtent.getWidth(),
-                    extentheight: mapExtent.getHeight(),
-                    vectorLayers: vectorLayers
-                };
-                var form = $('<form method="POST" action="' + url + '"  />');
-                $('<input></input>').attr('type', 'hidden').attr('name', 'data').val(JSON.stringify(data)).appendTo(form);
-                form.appendTo($('body'));
-                form.submit();
-                form.remove();
-            }
+            var form = $('<form method="POST" action="' + url + '"/>');
```

Comment:
I know you just transformed this from the old code, but we should really earmark this for a change.  
I suppose we could simply ajax the entire data as is, without using an ephemeral form at all?  
If not, at least attributes should be set with safe methods. Sth like
```
var $form = $('<form>');
$fom.attr('action', url);
$form.attr('method', 'POST');
```